tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py3{8,9,10,11,12,12-trio}, pep8, pypy3
+envlist = py3{8,9,10,11,12,12-trio,13}, pep8, pypy3
```

Comment:
there's no reason to leave trio at py312 specifically. When I added the test I simply elected to add it to the latest
```suggestion
# we only test trio on latest python version
envlist = py3{8,9,10,11,12,13,13-trio}, pep8, pypy3
```